.gitignore

```diff
@@ -9,3 +9,4 @@ cabal.sandbox.config
 *.sublime-*
 result
 *.code-*
+.ghc.environment.*
```

Comment:
What does this do?

Reply:
I am building the project with `cabal`, and it is creating such files locally, so it is a good idea to add these files to `gitignore` to not commit them. I can remove it if you don't expect the project to be built with anything but `nix` :+1:

Reply:
No, that's perfect then! I just didn't know because I build with `nix`. But if other people build with pure `cabal`, then this is valuable.